app/assets/stylesheets/base.scss

```diff
@@ -65,16 +65,18 @@ pre { word-wrap: break-word; }
 
 .back-to-top {
   background-color: $border-dark-grey;
-  border-radius: 10px;
+  border-radius: 4px;
   bottom: 20px;
   color: $white;
   display: block;
   font-size: 2.9em;
+  height: 50px;
   line-height: 1.5;
   opacity: 0;
-  padding: 0 12px;
   position: fixed;
   right: 54px;
+  transition: opacity ease 400ms;
+  width: 50px;
```

Comment:
https://github.com/diaspora/diaspora/blob/0f33a7e/.scss-lint.yml#L115 and https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#propertysortorder. Enforcing some order looked like a good idea and alphabetical order is the scss lint default. (also see the linked readme for more information) I don't have a strong opinion here but if you'd like to change this then let's discuss that somewhere else.